### PR TITLE
Translated all variables and function names in bot.py to English

### PR DESCRIPTION
**Pull request by MaryannAlonge**
Completed one of the tasks by carefully translating bot.py to English so that it can be easily understood by non-Spanish speakers.

All changes in one commit.
Avoided errors and inconsistencies.

### CODE REVIEW

File: data-gathering/bot.py [hunk 1]
```diff
@@ -19,62 +19,62 @@
...
-discussioneURL = urllib.parse.quote(discussione)
+sizeDiscussion = 1
+#discussion page prefix
+discussion = "Discussion:"
+discussionURL = urllib.parse.quote(discussion)
 
-#de momento, el conteo de avisos no funciona para la wikipedia en español
-configAvvisi = 0
+#At the moment, the notice count does not work for Wikipedia in Spanish
+configNotices = 0
```
Comment (wikicurricula-uy) [inline data-gathering/bot.py:45]: here you say configNotices

File: data-gathering/bot.py [hunk 2]
```diff
@@ -522,152 +522,152 @@ def analisi():
...
 
         if note:
 
-           ris = ris + note(wikitext) + "\t"
+           res = res + note(wikitext) + "\t"
 
            
 
-        if configAvvisi:
+        if configAlerts:
```
Comment (wikicurricula-uy) [inline data-gathering/bot.py:574]: and here you say configAlerts

**Review by wikicurricula-uy — CHANGES_REQUESTED** [review]
Hello! thanks for your work! you have an inconsistency in a variable name.
Please fix this.
To spot this kind of errors you should try running the script.
1. make sure query.csv contains a list of articles in the language you are using (now it has articles in spanish). I advice you cut the list at 10 items or so, so that excecution doesn't take too long
3. In your terminal, go to the data gathering folder
4. run `python3 bot.by`

If this fails, you probably made a mistake while translating.

Thanks!

File: data-gathering/bot.py (hunk 2 above)
Comment (MaryannAlonge) [reply data-gathering/bot.py:574]: hi @EneHacheCe @wikicurricula-uy this has now been fixed

**Comment by MaryannAlonge** [comment]
Hello @wikicurricula-uy @EneHacheCe I have now made the appropriate changes and pushed them in a new commit.

**Comment by wikicurricula-uy** [comment]
Hello, @MaryannAlonge ! I've tested the code and it seems to be working correctly now. Can you please have a shot at resolving the conflicts? If you gave trouble doing that please reach out to me (@EneHacheCe) or @fernandouval, we can help you.

Also, some comments that may improve this:
* if you create a results.tsv, the python code translate.py also needs to use results.csv instead of resultati.csv
* If possible, commit in this pull request only the translation (bot.py) and the files affected by it, I mean, do not include the readme 
* The correct translation for voce / voci is article / articles

**Comment by wikicurricula-uy** [comment]
Let's reopen this after fixing the conflicts

**Pull request closed without merging**